src/Microsoft.AspNet.Mvc.Xml/Resources.resx

```diff
@@ -120,6 +120,9 @@
   <data name="EnumerableWrapperProvider_InvalidSourceEnumerableOfT" xml:space="preserve">
     <value>The type must be an interface and must be or derive from '{0}'.</value>
   </data>
+  <data name="RequiredProperty_MustHaveDataMemberRequired" xml:space="preserve">
+    <value>DataContractSerializer does not recognize '{0}', so instead use '{1}' with 'IsRequired' set to 'True' for value type property '{2}' on type '{3}'.</value>
```

Comment:
Any programmatic name (like `DataContractSerializer`) should not be in the resource string.